core-server/src/main/java/org/glassfish/jersey/server/internal/monitoring/core/ReservoirConstants.java

```diff
@@ -16,28 +16,59 @@
 
 package org.glassfish.jersey.server.internal.monitoring.core;
 
+import java.security.PrivilegedAction;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 /**
  * The constants that determine the behaviour of sliding windows and their trimmers.
  *
  * @author Stepan Vavra
  */
 public final class ReservoirConstants {
 
+    /**
+     * JVM argument to define the value of {@link COLLISION_BUFFER_POWER}.
+     * Lower values reduce the memory footprint.
+     */
+    public static final String COLLISION_BUFFER_POWER_JVM_ARG = "monitoring.collision.buffer.power";
```

Comment:
Please move this to ServerProperties and use the usual "jersey.config.server" prefix.

Reply:
Done